setup.py

```diff
@@ -38,6 +38,11 @@
     raise ImportError('Please execute configure script first')
 
 
+libraries=['nvpair', 'zfs', 'zfs_core', 'uutil']
```

Comment:
Do you think it might be better to pass on the libraries from autoconf ? This way we only make changes in one place ?